content/en/docs/tasks/configure-pod-container/configure-runasusername.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Configure RunAsUserName for Windows pods and containers
+content_template: templates/task
+weight: 20
+---
+
+{{% capture overview %}}
+
+{{< feature-state for_k8s_version="v1.16" state="alpha" >}}
+
+This page shows how to enable and use the `RunAsUserName` feature for pods and containers that will run on Windows nodes. This feature is meant to be the Windows equivalent of the Linux' `runAsUser`, allowing users to run the container entrypoints with a different username that their default ones.
+
+{{< note >}}
+Currently this feature is in alpha state. The overall functionality of the feature will not change, but there may be some changes regarding the username validation. Please take this into consideration when testing or adopting this feature.
+{{< /note >}}
+
+{{% /capture %}}
+
+{{% capture prerequisites %}}
+
+You need to have a Kubernetes cluster and the kubectl command-line tool must be configured to communicate with your cluster. The cluster is expected to have Windows worker nodes where pods with containers running Windows workloads will get scheduled. This section covers the initial step required once for each cluster:
```

Comment:
Recommend removing the sentence "This section covers the initial step required once for each cluster:" The next paragraph makes that clear.

Reply:
Done